content/articles/email-validator-in-flutter/index.md

```diff
@@ -0,0 +1,238 @@
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+```
+
+This is where all the code we need to run for the page is. This page contains text field, button labels, and widgets.
+
+- Step 3: Second screen.
+Create a new file in the lib folder and call it `secondpage.dart`. Then, in the same file, add the snippets below.
+
+```dart
+//import are here
+import 'package:flutter/material.dart';
+class SecondScreen extends StatelessWidget {
+  const SecondScreen({Key? key}) : super(key: key);
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(),
+      body: const Center(
+        child: Text('Welcome to the Second Page'),
+      ),
+    );
+  }
+}
+```
+
+- Step 4 - Testing the application
+We need to confirm if the application works perfectly and our validator works as desired. You need to ensure you set up an emulator or a mobile device set up to run the application.
+
+Execute the command below to run the application.
+
+```bash
+flutter run 'name-of-the-device'
+```
+
+or you can run using a browser
+
+```bash
+flutter run -d
+```
+
+Below are screenshots showing the expected results.
+
+|                                        Trying login without filling in any detail                                        |                                        inValid email input                                        |
+| :----------------------------------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------: |
+| ![Trying login without filling in any detail](/engineering-education/flutter-email-validator/missingemail.jpg) | ![inValid email input](/engineering-education/flutter-email-validator/invalidemail.jpg) |
+
+|                                       Valid email input                                       |                                         After running and valid email                                         |
+| :-------------------------------------------------------------------------------------------: | :-----------------------------------------------------------------------------------------------------------: |
+| ![Valid Email Input](/engineering-education/flutter-email-validator/validemail.jpg) | ![After running and valid email](/engineering-education/flutter-email-validator/validatedemail.jpg) |
+
+### Conclusion
+Email validator is one of the best ways to confirm emails in Flutter. It helps to reduce the invalid emails that users may use to sign up and help one easily maintain the available emails.
+
+I recommend any programmer to use any flutter application due to its' compatibility with all levels of flutter application, be it flutter web application, flutter mobile application, or any other application.
```

Comment:
A conclusion should summarize your article as well. What can be learned from your article is the conclusion here.